pipeline-model-api/pom.xml

```diff
@@ -40,19 +40,18 @@
   <dependencies>
     <!-- JSON schema stuff -->
     <dependency>
-      <groupId>org.jenkins-ci.lib</groupId>
-      <artifactId>pipeline-model-json-shaded</artifactId>
-      <version>${project.version}</version>
-      <exclusions>
-        <exclusion>
-          <groupId>com.fasterxml.jackson.core</groupId>
-          <artifactId>jackson-databind</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>com.github.fge</groupId>
-          <artifactId>json-schema-validator</artifactId>
-        </exclusion>
-      </exclusions>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>jackson2-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.datatype</groupId>
+      <artifactId>jackson-datatype-json-org</artifactId>
+      <version>2.8.11</version>
```

Comment:
Can you omit the version to get it automatically from the parent's `dependencyManagement`?

Reply:
Whoops, forgot to do that. Fixing.